lib/matplotlib/transforms.py

```diff
@@ -115,7 +115,7 @@ def __getstate__(self):
     def __setstate__(self, data_dict):
         self.__dict__ = data_dict
         # turn the normal dictionary back into a WeakValueDictionary
-        self._parents = WeakValueDictionary(self._parents)
+        self._parents = WeakValueDictionary(self.__dict__['_parents'])
```

Comment:
I still don't understand... shouldn't `self._parents` be equivalent to
`self.__dict__['_parents']`, since the object dictionary would be where one
accesses attributes of the object?

On Fri, Feb 6, 2015 at 11:16 AM, Phil Elson <notifications@github.com>
wrote:

> In lib/matplotlib/transforms.py
> <https://github.com/matplotlib/matplotlib/pull/4070#discussion_r24250906>:
>
> > @@ -115,7 +115,7 @@ def __getstate__(self):
> >      def __setstate__(self, data_dict):
> >          self.__dict__ = data_dict
> >          # turn the normal dictionary back into a WeakValueDictionary
> > -        self._parents = WeakValueDictionary(self._parents)
> > +        self._parents = WeakValueDictionary(self.__dict__['_parents'])
>
>  I don't think this is doing what you thing it is.
>
> Scrap that. I think I'm talking rubbish. (Friday afternoon!)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4070/files#r24250906>.
>